sphinx/themes/basic/static/doctools.js

```diff
@@ -232,24 +229,24 @@ const Documentation = {
             const prevLink = document.querySelector('link[rel="prev"]');
             if (prevLink && prevLink.href) {
               window.location.href = prevLink.href;
-              return false;
+              event.preventDefault();
             }
             break;
           case "ArrowRight":
             if (!DOCUMENTATION_OPTIONS.NAVIGATION_WITH_KEYS)
               break;
 
-            const nextLink = document.querySelector('link[rel="next"]').href;
+            const nextLink = document.querySelector('link[rel="next"]');
```

Comment:
Looks like this one is being fixed in https://github.com/sphinx-doc/sphinx/pull/10243